### ISSUE TITLE
Toolbar Color Crash When Fast Scroll Recyclerview 

### ISSUE THREAD

**Issue by sajednt** [florent37/MaterialViewPager#144]
hi ,

when i set the background color and image url by setMaterialViewPagerListener , and fast scroll recyclerview in one of viewpagers , toolbar color in this way comes

![screenshot_2015-08-15-18-16-56](https://cloud.githubusercontent.com/assets/8393748/9289128/161017f4-437b-11e5-880b-548b340923a6.png)
![screenshot_2015-08-15-18-16-52](https://cloud.githubusercontent.com/assets/8393748/9289130/19410dc0-437b-11e5-805b-af265628514d.png)

this issue is very important for me !
please help me to fix this issue :(

**Comment by wsw87201326** [comment]
yes,i also hava this question.when i set app:viewpager_hideToolbarAndTitle="false",the title weill not hide some time , and alse show when i slide.

**Comment by krzysiek-icomapss** [comment]
problem: method setColor() in MaterialVIewPagerAnimator there is animation using parameter lastPercent which is calculated from scroll offset. You can change this to 0 and the problem will be resorved but than when u will change page and mini toolbar with tab is showing than this tolbar will be with alpha 0 (transparent). So pls check lastPercent parameter in first situaton and another and than maybe after checking valu of this parameter in one situation use this parameter and in another use 0.
